tests/test_pr_2D_TR_directional_sensors.py

```diff
@@ -71,24 +72,24 @@ def test_pr_2D_TR_directional_sensors():
     input_filename = f'example_tr_dir_input.h5'
     pathname = gettempdir()
     input_file_full_path = os.path.join(pathname, input_filename)
-    input_args = {
-        'pml_inside': False,
-        'pml_size': PML_size,
-        'smooth_p0': False,
-        'save_to_disk': True,
-        'input_filename': input_filename,
-        'data_path': pathname,
-        'save_to_disk_exit': True
-    }
-
-    # run the simulation for omnidirectional detector elements
-    kspaceFirstOrder2DC(**{
-        'medium': medium,
-        'kgrid': kgrid,
-        'source': deepcopy(source),
-        'sensor': deepcopy(sensor),
-        **input_args
-    })
+    input_args = SimulationOptions(
```

Comment:
could it be clearer to call this object sim_opts?

Reply:
Agreed and renamed to `simulation_options` to be more explicit